Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.21.0
+COPY . /httpRelay
+WORKDIR /httpRelay
+RUN make all
+
+FROM golang:1.21.0-alpine
```

Comment:
You don't need Go support anymore, as you have built the binaries at this point and this is the base image for executing the binaries. Isn't there a Alpine linux image that is smaller than this one?

Reply:
Agree with your opinion, changed to `alpine:3.18.3` Now the built image size reduce to 21.8MB from 236MB.